test/back_end/api_get_test.spec.js

```diff
@@ -3,254 +3,294 @@ const chai = require("chai");
 const chaiHttp = require("chai-http");
 chai.use(chaiHttp);
 const expect = chai.expect;
+const base64o = require("../../src/base64js.min.js");
+const base64 = base64o.base64
 
-describe("Test GET requests to API", function() {
-    it("should return user info for prof1", async function() {
-        let fetched = await fetch(
-            "http://localhost:3000/get_info.php?what=user_info&user_id=prof1"
-        ).catch(function(err) {
-            throw err;
+describe("Test POST/UPDATE/DELETE requests to API", function() {
```

Comment:
This is great, but you edited `api_get_test`. You were supposed to create `api_post_test`